lib/diagnostic.rb

```diff
@@ -6,37 +6,47 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'ruby lib/example.rb'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'pry'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'variable = "star wars episode viii"'
 
 # String interpolation in Ruby
 #
 # Use string interpolation to complete the sentence below. Replace "ANSWER" with
 # the correct code.
-
+answer = 'ANSWER'
 character = 'Jar Jar Binks'
+character.prepend('is really a sith lord')
+answer.delete!('ANSWER')
+
+
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = "'character' + 'is really a Sith Lord.'"
 
```

Comment:
If ` character = 'Jar Jar Binks'`, to interpolate it in a string, we'd just need to do `"#{character} is really a sith lord"`